src/main/java/com/direwolf20/buildinggadgets/common/helpers/NBTHelper.java

```diff
@@ -0,0 +1,13 @@
+package com.direwolf20.buildinggadgets.common.helpers;
+
+import net.minecraft.nbt.INBT;
+import net.minecraft.nbt.ListNBT;
+
+import java.util.stream.Collector;
+
+public final class NBTHelper {
```

Comment:
No thanks, lets avoid this crap :+1: I want as little helper classes as I can and this has been added with only one use.